cli/README.md

```diff
@@ -353,6 +352,10 @@ If no npm project is found, it wil run `npm init -y` in the folder where it is i
 
 ### Install @prisma/client
 
-After identifying the npm project, the generator will ensure that the `@prisma/client` package is installed. If it is already present in the `package.json` file, no action is taken. Otherwise [Joël - explain]
+After identifying the npm project, the generator will ensure that the `@prisma/client` package is installed. If it is already present in the `package.json` file, no action is taken. Otherwise it will run `npm install @prisma/client` to add the package as a dependency.
```

Comment:
What about the case if there is no `package.json`?

Reply:
It's covered in the point before in `### Identifying the npm project`
`If no npm project is found, it wil run `npm init -y` in the folder where it is invoked to create a new npm project.`